modules/update-service-mirror-release.adoc

```diff
@@ -0,0 +1,211 @@
+[id="update-service-mirror-release_{context}"]
+= Mirroring the {product-title} image repository
+
+The OpenShift Update Service requires a locally accessible registry containing update release payloads. 
+
+[IMPORTANT]
+====
+To avoid excessive memory usage by the OpenShift Update Service application the release images should be mirrored to a separate repository. To accomplish this you must follow the instructions below.
+====
+
+.Prerequisites
+
+* Review and complete the steps from xref:../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Creating a mirror registry for installation in a restricted network] up to, but not including, section *Mirroring the {product-title} image repository*.
+* Your mirror host has access to the Internet.
+* You configured a mirror registry to use in your restricted network and
+can access the certificate and credentials that you configured.
+ifndef::openshift-origin[]
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site and modified it to include authentication to your mirror repository.
+endif::[]
+ifdef::openshift-origin[]
+* You have created a pull secret for your mirror repository.
+endif::[]
+* If you use self-signed certificates that do not set a Subject Alternative Name, you must precede the `oc` commands in this procedure with `GODEBUG=x509ignoreCN=0`. If you do not set this variable, the `oc` commands will fail with the following error:
++
+[source,terminal]
+----
+x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
+----
+
+.Procedure
+
+Complete the following steps on the mirror host:
+
+. Review the
+link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
+to determine the version of {product-title} that you want to install and determine the corresponding tag on the link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Repository Tags] page.
```

Comment:
The version should be the one you want to upgrade to. So I would suggest to replace install with upgrade to.

Reply:
done